middleware/kubernetes/setup.go

```diff
@@ -98,6 +98,27 @@ func kubernetesParse(c *caddy.Controller) (*Kubernetes, error) {
 						continue
 					}
 					return nil, c.ArgErr()
+				case "certauth":
```

Comment:
We should keep this in sync with `middleware/etcd/setup.go` where we've usd `tls <something>` to specify the tls parameters.